xtas/core.py

```diff
@@ -68,8 +68,8 @@ def configure(config, import_error="raise", unknown_key="raise"):
             else:
                 logger.warn(msg)
 
-    app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
```

Comment:
Is there a reason to use a string here, with xtas namespace, but the value directly on the next line?